I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 8, in script
    init python:
  File "game/script.rpy", line 12, in <module>
    from firebase_fetcher import get_emotion
ModuleNotFoundError: No module named 'pyrebase'

-- Full Traceback ------------------------------------------------------------

Traceback (most recent call last):
  File "game/script.rpy", line 8, in script
    init python:
  File "renpy/ast.py", line 1187, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/python.py", line 1260, in py_exec_bytecode
    exec(bytecode, globals, locals)
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "game/script.rpy", line 12, in <module>
    from firebase_fetcher import get_emotion
  File "firebase_fetcher.py", line 2, in <module>
    import pyrebase
ModuleNotFoundError: No module named 'pyrebase'

Windows-11-10.0.26100-SP0 AMD64
Ren'Py 8.4.1.25072401
SpeechPerfect 1.0
Tue Oct  7 23:55:46 2025
